### DISCUSSION TITLE
Mosquitto Installazione Windows

### DISCUSSION THREAD

**Discussion by omarantelmi** [indomus/forum#2042] [Networking]
Salve
Sto seguendo la vs. guida per installare Mosquitto su Windows (windows 11 in questo caso specifico) e purtroppo mi sono imbattuto in alcuni problemi

1) Indicate di Scaricare l’ultima “binary version – Cygwin” ma se accedo al link della guida, trovo la binary version (al momento 2.0.15) ma non è indicato da nessuna parte "Cygwin"
2) Installo comunque questa versione, ma alla fine dell'installazione non esce alcun errore
3) Procedo con la guida installando OPEN SSH (Win64 OpenSSL v3.0.7 - NO versione LITE) tramite il link fornito da voi sempre nella guida, installo la versione a 64Bit in quanto la macchina è 64Bit, ma una volta terminata l'installazione nella cartella di OPEN non trovo i DLL indicati

Come potrei risolvere?
Grazie per tutto il lavoro che fate!

P.s. non so sinceramente se la sezione è giusta, ma non sapevo dove meglio categorizzarla

**Comment by snakuzzo** [comment]
Consiglio...lascia perdere questo tipo di installazione. Con Docker tutti questi problemi non li avrai

**Reply by omarantelmi** [reply]
C'è una guida per Mosquitto e docker su Windows? 

Questa per caso? https://indomus.it/guide/installare-e-configurare-il-broker-mqtt-mosquitto-su-docker-su-linux-debian/

**Reply by snakuzzo** [reply]
Per l'installazione di docker non è che serva chissà quale guida. Scarichi, Clicchi. Installi. Comunque puoi trovare [QUI](https://indomus.it/guide/come-installare-e-configurare-home-assistant-core-con-docker-su-windows/) come si fa.

Per mosquitto poi, basta che leggi [QUI](https://indomus.it/guide/come-installare-e-configurare-il-broker-mqtt-mosquitto-su-docker-su-raspbian/#mosquitto)...a parte i path scritti in modo diverso, non cambia nulla

**Reply by omarantelmi** [reply]
Risolto! Grazie mille per l'aiuto!
